docs/calicoctl/checksystem.md

```diff
@@ -0,0 +1,55 @@
+
+# User guide for 'calicoctl checksystem' commands
+
+This section describes the `calicoctl checksystem` commands.
+
+The `calicoctl checksystem` command allows users to check for 
+incompatibilties between Calico and the host system.
+
+Read the [calicoctl command line interface user guide](../calicoctl.md) for a full list of calicoctl commands.
+
+## Displaying the help text for 'calicoctl checksystem' commands
+
+Run `calicoctl checksystem --help` to display the following help menu for the 
+calicoctl checksystem commands.
+
+```
+
+Usage:
+  calicoctl checksystem
+
+Description:
+  Check for incompatibilities between Calico and the host system
+
+Options:
+  --fix  DEPRECATED: checksystem no longer fixes issues that it detects
+  --libnetwork  Check for the correct docker version for libnetwork deployments
+```
+
+## calicoctl checksystem commands
+
+### calicoctl checksystem 
+This command allows you to verify that your host system is configured correctly
+for calicoctl to manage a Calico network.
```

Comment:
It would be good to describe in general terms what kinds of things are being checked, e.g. iptables and ipsets support, IP forwarding, etc.